src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder stringBuilder = new StringBuilder();
+        text = text.toLowerCase().replaceAll("[^a-z0-9]","");
```

Comment:
```suggestion
        text = text.toLowerCase().replaceAll("[^a-z0-9]", "");
```
don't forget about spaces